Unit test failing in Windows OS for cli runtime wait.go

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may delay the bug being addressed. Thanks!
-->

### Please fill out the issue checklist below and provide ALL the requested information.

- [x] I reviewed open and closed Github issues that may be related to my problem.
- [x] I am reporting a bug that others will be able to reproduce.

### Describe the bug
Unit test failing in Windows OS for wait.go file test cases
OS: Windows
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
    	panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x112c7a2]


### Expected behavior
All unit test cases are successful 

### Steps to Reproduce
Steps to reproduce the behavior:
clone apps-cli-plugin code git clone https://github.com/vmware-tanzu/apps-cli-plugin in windows and run go test ./....
### Screenshots
If applicable, add screenshots to help explain your problem.

### Version (Apps plugin version, Version of K8s running on cluster)
Put the output of the following commad
```sh
tanzu version && tanzu apps version
``` 

### Environment where the bug was observed (cloud, OS, etc)

### Additional context & Relevant Debug Output (Logs, etc)
Add any other context about the problem here.

--- a/pkg/cli-runtime/wait/wait_test.go
+++ b/pkg/cli-runtime/wait/wait_test.go
@@ -158,7 +158,7 @@ func TestUntilDelete(t *testing.T) {
 	defer func() {
 		BackOffTime = previousBackOffTime
 	}()
-	BackOffTime = 10 * time.Millisecond
+	BackOffTime = 30 * time.Millisecond
 	workload := &cartov1alpha1.Workload{
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: defaultNamespace,
@@ -189,7 +189,7 @@ func TestUntilDelete(t *testing.T) {
 		reactorFunc clitesting.ReactionFunc
 	}{{
 		name:    "smaller timeout",
-		timeout: time.Millisecond,
+		timeout: time.Nanosecond,
 		err:     context.DeadlineExceeded,
 	}, {
 		name:    "API not found error",

